docs/language-server/project-manager-http-endpoints.md

```diff
@@ -17,9 +17,11 @@ JSONRPC protocol.
 
 <!-- /MarkdownTOC -->
 
-## `/projects/{project_id}/enso-project`
+## `/projects/{project_id}/enso-project?projectsDirectory={projects_path}`
 
-HTTP endpoint that returns the project structure in `.enso-project` format.
+HTTP endpoint that returns the project structure in `.enso-project` format. The
+optional `projectsDirectory` parameter allows the user to specify a custom path
+to projects' directory (e.g. if it is in a subfolder).
```

Comment:
How is it guaranteed, somebody will not get access to content of my `/etc` (or any other) directory that isn't holding Enso projects?

Reply:
As long as `/etc` doesn't contain any projects, nothing is returned in the response. Moreover, you need to know the correct project id to access the project.